### ISSUE TITLE
Thumbnail index is not keyboard/screen-reader accessible (media viewer)

### ISSUE THREAD

**Issue by jvine** [sul-dlss/sul-embed#669]
(used to be fine; got broken somewhere; media didn't break it as it affects images, too)

Currently the "..." icon and the list items don't get keyboard focus. And once the index is open, the list items don't get keyboard focus either.

Here's a model that is working:
https://searchworks.stanford.edu/?utf8=%E2%9C%93&search_field=search&q=5762702
(the green Online button opens a list of links)

- the div that contains the "..." icon should be a `<button>` element
- the aria-expanded attribute is not set correctly:  it says aria-expanded="true" all the time. it should toggle true/false as the div is opened/closed
- Bootstrap (and SearchWorks) has the aria-expanded attribute on both the parent button and the child div - it's not clear to me that both are necessary, but it doesn't hurt anything - so I'm guessing we should add it to the thumb-slider div and toggle it to match the "..." button
- when the thumb-slider is open, the list items within it don't get focus. The divs should be (or contain)  `<a>` elements with an href or tabindex="0".

**Comment by ndushay** [comment]
hrmm - we need to look at https://jirasul.stanford.edu/jira/browse/MEDIAINFRA-44  and determine what work remains.   @hannahfrost @jkeck

**Comment by ndushay** [comment]
@jvine   help!  I'm trying to allow each list element in the thumb-slider to get focus.  I've tried a bunch of things, but only the active element gets focus in Firefox.  

![list element](https://cloud.githubusercontent.com/assets/96775/18108419/f7ee95b2-6ec0-11e6-9290-5637c643c64a.png)

That works in Chrome, but not FF.  In addition to `href="javascript:;"` I've tried   `href="0"`, `tabindex="0"`, `tabindex="n"` (where n is the index to the list) ...   but nothing seems to work in FF.

**Comment by ndushay** [comment]
- [x]  the div that contains the "..." icon should be a `<button>` element - done.
- [x] add aria-expanded attribute to the thumb-slider div 
- [ ] when the thumb-slider is open, the list items within it don't get focus. The divs should be (or contain)  `<a>` elements with an href or tabindex="0".  

probably will be a separate ticket
- [ ] the aria-expanded attribute is not set correctly:  it says aria-expanded="true" all the time. it should toggle true/false as the div is opened/closed;  same for thumb-slider div aria-expanded attr.

**Comment by jvine** [comment]
This is primarily Firefox's "fault" (technically they are following the OS rules but it's stupid anyway). If you go to keyboard settings and click All controls under Shortcuts, FF will allow you to tab to links. 

![screen shot 2016-08-30 at 3 27 19 pm](https://cloud.githubusercontent.com/assets/6945691/18109667/a8c7afda-6ec7-11e6-80a9-659c90d73ee7.png)

Using `<button>` will override this, but 
a) using `<a>` will put the list of image titles in the links list for the page (good), and 
b) any keyboard-only Mac FF user will know to configure their keyboard.

For the `<a>` it appears that `<a href="#">` is the best practice and doesn't require tabindex.
